index-fr.html

```diff
@@ -14,80 +14,91 @@
     <h1>Code de conduite pour la conférence</h1>
 
     <p>
-      Tous les participants, conférenciers, sponsors et volontaires pour notre
-      conférence doivent accepter le code de conduite suivant. Les organisateurs
-      s'attacheront à faire respecter ce code durant l'événement. Nous attendons
-      de la part de chaque participant une coopération pour assurer un
-      environnement sain pour tous.
+      Toutes les personnes participant à la conférence,
+      y compris participant·e·s, orateurs·trices, sponsors et volontaires
+      s'engagent à accepter le code d'éthique et de déontologie, ou "code de conduite" suivant.
```

Comment:
```suggestion
      s'engagent à accepter le code d'éthique et de déontologie, ou « code de conduite » suivant.
```

Guillemets français :wink: